public/app/plugins/panel/nodeGraph/Edge.tsx

```diff
@@ -1,16 +1,21 @@
 import React, { MouseEvent, memo } from 'react';
 
-import { nodeR } from './Node';
+import { EdgeArrowMarker } from './EdgeArrowMarker';
+import { computeNodeCircumferenceStrokeWidth, nodeR } from './Node';
 import { EdgeDatum, NodeDatum } from './types';
 import { shortenLine } from './utils';
 
+export const highlightedEdgeColor = '#a00';
```

Comment:
Does this work well with the light theme? I ideally I think we should pick some color from theme so that it gets adjusted according to theme.